test/e2e/volumes.go

```diff
@@ -76,7 +75,6 @@ func DeleteCinderVolume(name string) error {
 	return err
 }
 
-
 // These tests need privileged containers, which are disabled by default.  Run
 // the test with "go run hack/e2e.go ... --ginkgo.focus=[Feature:Volumes]"
 var _ = framework.KubeDescribe("Volumes [Feature:Volumes]", func() {
```

Comment:
Can you change the describe text to "Volumes [Volume]"? Reason:
```
grep -Rn "Feature:Volumes" test/
test/e2e/common/volumes.go:345:// the test with "go run hack/e2e.go ... --ginkgo.focus=[Feature:Volumes]"
test/e2e/volumes.go:385:// the test with "go run hack/e2e.go ... --ginkgo.focus=[Feature:Volumes]"
test/e2e/volumes.go:386:var _ = framework.KubeDescribe("Volumes [Feature:Volume]", func() {
```
grep -Rn "\[Volume\]" test/ |wc -l
120
```